updatehub/src/http_api.rs

```diff
@@ -44,8 +44,9 @@ impl API {
 
     async fn probe(
         agent: web::Data<API>,
-        server_address: Option<String>,
+        server_address: String,
```

Comment:
The commit log does not explain what issues does this fix. Do you mind to rework it?

Reply:
I've updated the commit message.